src/network/connection.cpp

```diff
@@ -832,62 +831,30 @@ PeerHelper::PeerHelper() :
 PeerHelper::PeerHelper(Peer* peer) :
 	m_peer(peer)
 {
-	if (peer != NULL)
-	{
-		if (!peer->IncUseCount())
-		{
-			m_peer = 0;
-		}
-	}
+	if (!peer) return;
```

Comment:
Please don't add conditional bodies on the same line as the statement unless you're trying to hide something... this violates the Linux code style guidelines anyway (which ours inherit)